drupal/composer.json

```diff
@@ -120,12 +120,12 @@
         "pantheon-systems/drupal-integrations"
       ],
       "locations": {
-          "web-root": "./web"
+        "web-root": "./web"
       },
       "file-mapping": {
-          "[project-root]/.editorconfig": false,
```

Comment:
@alex-ahumada It looks like a few unrelated changes got in. Can we fix those please?

Sorry for the late review.

Reply:
Done @shadcn! Sorry, my WebStorm config removed some whitespace.